types/transaction.go

```diff
@@ -74,6 +74,14 @@ func (t *Transaction) IsContractCreation() bool {
 	return t.To == nil
 }
 
+// IsValueTransfer checks if tx is a value transfer
+func (t *Transaction) IsValueTransfer() bool {
+	return t.Value != nil &&
+		t.Value.Sign() != 0 &&
```

Comment:
Do we even need to check the sign?
Otherwise, I would check if it is greater than 0.

```suggestion
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1998/commits/098fffb88db91e73bbd05d4215e8edad1bb00d80